crypto/ml_kem/ml_kem_ipd_ref_common/indcpa.c

```diff
@@ -19,12 +22,13 @@
 *              polyvec *pk: pointer to the input public-key polyvec
 *              const uint8_t *seed: pointer to the input public seed
 **************************************************/
-static void pack_pk(uint8_t r[KYBER_INDCPA_PUBLICKEYBYTES],
+static void pack_pk(ml_kem_params *params,
+                    uint8_t *r,
                     polyvec *pk,
-                    const uint8_t seed[KYBER_SYMBYTES])
+                    const uint8_t *seed)
```

Comment:
Isn't `seed` always `KYBER_SYMBYTES = 32` bytes long, irrespective of parameter set? And we use exactly `KYBER_SYMBYTES` bytes of `seed`, so seems worth keeping the constraint.